gapis/api/service.proto

```diff
@@ -89,6 +89,22 @@ enum FramebufferAttachment {
   Color3 = 5;
 }
 
+enum FramebufferAttachmentType {
+  DepthAttachment = 0;
+  ColorAttachment = 1;
+}
+
+message FramebufferAttachments {
+  repeated FramebufferAttachmentVulkan attachments = 1;
+}
+
+message FramebufferAttachmentVulkan {
```

Comment:
no Vulkan.

Reply:
This is partially why it's WIP-do-not-review. This was a temp name